conf/init/nginx_conf_create.py

```diff
@@ -124,6 +124,21 @@ def generate_rate_limiting_config(config):
     )
 
 
+def generate_http_base_config(config):
+    """
+    Generates http base config from the app config.
+    """
+    config = config or {}
+    feature_proxy_protocol = config.get("FEATURE_PROXY_PROTOCOL", False)
```

Comment:
proxy protocol is always enabled. The change we want to make is to see which header is our source of truth for the IP chain, I don't think we ever need to disable this

Reply:
Proxy protocol is not always enabled. If a load balancer doesn't support it, you should not be using Quay's  port dedicated for it (7443). I don't think that would work at all. X-Forwarded-For is done through HTTP header, while proxy protocol is TCP metadata that nginx knows how process.